src/app/content/components/CallToActionPopup/styles.ts

```diff
@@ -0,0 +1,124 @@
+import styled from 'styled-components';
+import { Times } from 'styled-icons/fa-solid/Times/Times';
+import Button from '../../../components/Button';
+import { textRegularStyle } from '../../../components/Typography';
+import { textStyle } from '../../../components/Typography/base';
+import theme from '../../../theme';
+
+// tslint:disable-next-line: variable-name
+export const CTAWrapper = styled.div`
+  position: fixed;
+  left: 0;
+  bottom: 0;
+  z-index: 99;
+  width: 100%;
+
+  @media screen and (max-width: 1200px) {
+    display: none;
+  }
+`;
+
+// tslint:disable-next-line: variable-name
+export const CTAContent = styled.div`
+  position: absolute;
+  z-index: 3;
+  bottom: 0;
+  width: 100%;
+  display flex;
+  justify-content: center;
+  align-items: center;
+`;
+
+// tslint:disable-next-line: variable-name
+export const CTAText = styled.div``;
```

Comment:
pretty sure this'll throw some lint errors or something, if this doesn't need to be a styled component you might want to just use an inline div